src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -962,6 +962,7 @@ protected function createChildContext(array $parentContext, string $attribute, ?
 
     public function testChildContextKeepsOriginalContextCacheKey()
     {
+        $this->markTestSkipped('Scalar attributes change context. See https://github.com/symfony/symfony/pull/58473');
```

Comment:
Having a test that is always skipped does not make sense. Let’s rewrite it in a meaningful way instead.